### PR TITLE
add edit in kebab dropdown of address space detail header and purge option in address detail header

### PR DESCRIPTION
**Pull request by drcoolsanjeev**
### Type of change

- Enhancement / new feature

### Description
- add download option in kebab of address space list
- add edit option in kebab of address space detail header
- add purge option in kebab of address detail header

### CODE REVIEW

File: console/console-init/ui/src/pages/AddressSpaceDetail/AddressSpaceDetailPage.tsx [hunk 1]
```diff
@@ -30,12 +32,16 @@ import { useHistory } from "react-router";
...
+  EDIT_ADDRESS_SPACE
 } from "queries";
 import { DialoguePrompt } from "components/common/DialoguePrompt";
-import { POLL_INTERVAL } from "constants/constants";
+import { POLL_INTERVAL, FetchPolicy } from "constants/constants";
 import { ErrorAlert } from "components/common/ErrorAlert";
 import { NoDataFound } from "components/common/NoDataFound";
+import { EditAddressSpace } from "pages/EditAddressSpace";
+import { IAddressSpace } from "components/AddressSpaceList/AddressSpaceList";
+import { string } from "prop-types";
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/pages/AddressSpaceDetail/AddressSpaceDetailPage.tsx]: Unused variable.

File: console/console-init/ui/src/components/AddressSpace/AddressSpaceHeader.tsx [hunk 2]
```diff
@@ -19,6 +19,7 @@ import {
 import { css, StyleSheet } from "@patternfly/react-styles";
 import { AddressSpaceType } from "components/common/AddressSpaceListFormatter";
 import { FormatDistance } from "use-patternfly";
+import { IAddressSpace } from "components/AddressSpaceList/AddressSpaceList";
```
Comment (rkpattnaik780) [inline console/console-init/ui/src/components/AddressSpace/AddressSpaceHeader.tsx]: Unused vaariable.

**Comment by drcoolsanjeev** [comment]
@enmasse-ci run tests testcase=standard.web.*

**Comment by kornys** [comment]
@enmasse-ci run tests testcase=standard.web.**

**Comment by kornys** [comment]
@drcoolsanjeev so Ive just tried your changes and I see that when I open edit menu win address menu it has set wrong addressplan

**Comment by drcoolsanjeev** [comment]
@kornys , I fixed the issue. Please review the changes.

File: console/console-init/ui/src/pages/AddressSpaceDetail/AddressSpaceDetailPage.tsx [hunk 3]
```diff
@@ -168,6 +209,53 @@ export default function AddressSpaceDetailPage() {
...
+    }
+  };
+
+  const handleCancelEdit = () => {
+    setIsEditModalOpen(false);
+    setAddressSpaceBeingEdited(null);
+  };
+  const handleSaving = async () => {
+    addressSpaceBeingEdited &&
+      (await client.mutate({
```
Comment (ajaypratap003) [inline console/console-init/ui/src/pages/AddressSpaceDetail/AddressSpaceDetailPage.tsx]: we are using custom hook for mutation query i.e. useMutationQuery  for server error handling.
Replace this code with useMutaionQuery hook as we implemented in other page/form.

**Pull request merged**